nimble/controller/src/ble_ll_scan.c

```diff
@@ -2447,8 +2445,13 @@ ble_ll_hci_send_ext_adv_report(uint8_t ptype, uint8_t *adva, uint8_t adva_type,
 
     BLE_LL_ASSERT(offset <= datalen);
 
-    /* Store any event left for later use */
-    aux_data->evt = evt;
+    if (aux_data){
+        /* Store any event left for later use */
+        aux_data->evt = evt;
+    } else {
+        /* If it is empty beacon, evt shall be NULL */
+        BLE_LL_ASSERT(!evt);
```

Comment:
`BLE_LL_ASSERT(evt)` ?

Reply:
evt shall be NULL so  this check is good.